bcda/web/router.go

```diff
@@ -34,6 +34,14 @@ func NewAPIRouter() http.Handler {
 		r.With(auth.RequireTokenAuth, auth.RequireTokenJobMatch).Get(m.WrapHandler("/jobs/{jobID}", jobStatus))
 		r.Get(m.WrapHandler("/metadata", metadata))
 	})
+
+	if utils.GetEnvBool("VERSION_2_ENDPOINT_ACTIVE", false) {
+		r.Route("/api/v2", func(r chi.Router) {
+			r.With(auth.RequireTokenAuth, ValidateBulkRequestHeaders).Get(m.WrapHandler("/Patient/$export", bulkPatientRequestV2))
```

Comment:
If we decide on moving out the v2 into a separate package, we can use named imports to help with the package name collision.

So we could have the following:
```
import (
...
webV2 github.com/CMSgov/bcda-app/bcda/web/v2
...
)
```

And then we would use `webV2.BulkPatientRequestV2`